src/btree.rs

```diff
@@ -3,15 +3,13 @@ use crate::oram::BUCKET_SIZE;
 #[derive(Debug, Clone, Default)]
 pub struct BTree {
     pub(super) root: Option<Box<Node>>,
-    nb_blocks: usize,
     height: u16,
 }
 
 impl BTree {
     pub fn init_new(dummies: &mut Vec<DataItem>, nb_blocks: usize) -> BTree {
         let mut tree = BTree {
             root: Option::None,
-            nb_blocks,
             height: nb_blocks.ilog2() as u16 + 1,
```

Comment:
No, since a complete tree has `2^x - 1` nodes and has a height of `x`. If you init BTree with a power of 2 as the number of nodes, then a ceiling function will not "plus one" to the expected value, since 2^x nodes result in an incomplete binary tree with only one node on the last level.

A floor function with + 1 is in my opinion the best thing to do.